array-manipulation-3/src/union.js

```diff
@@ -0,0 +1,14 @@
+/* exported union */
+function union(first, second) {
+  var combined = first;
```

Comment:
This is not actually making a copy, this is just storing that same array in an additional variable. If you make changes to the array in this variable, you are actually changing the original. In order to make an easy copy of an array, You can use the `slice` method of arrays. If given no arguments, it will just make a copy of the entire array. Make sure you take a look at the docs to fully understand its purpose, though!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice